requirements.txt

```diff
@@ -87,25 +85,24 @@ python-json-logger==2.0.7
 python-magic==0.4.27
 pytz==2023.3
 PyYAML==5.4.1
-redis==4.5.4
-requests==2.28.2
+redis==4.5.5
+requests==2.31.0
 rfc3339-validator==0.1.4
-s3transfer==0.6.0
-sentry-sdk==1.20.0
+s3transfer==0.6.1
+sentry-sdk==1.25.1
 six==1.16.0
 smartypants==2.0.1
-SQLAlchemy==1.4.47
+SQLAlchemy==1.4.48
 statsd==4.0.1
-tenacity==8.2.2
-twilio==8.0.0
-typing_extensions==4.5.0
+twilio==8.3.0
+typing_extensions==4.6.3
 Unidecode==1.3.6
-urllib3==1.26.15
+urllib3==1.26.16
```

Comment:
I thought we also couldn't upgrade urllib3 due to the alpine container version - was this code deployed to dev?